se-thaumatin/processing_in_detail.md

```diff
@@ -364,7 +364,7 @@ should not be a cause for concern.
 Once you have indexed the data you may optionally attempt to infer the
 correct Bravais lattice and assign this to constrain the unit cell in
 subsequent processing. If, for example, the unit cell from indexing
-has all three angles close to 90 degrees and two unit cell lengths
+has all three angles close to 90° and two unit cell lengths
```

Comment:
I was working in an emacs buffer at the time so couldn't be bothered to copy / paste in